cli/test/smokehouse/test-definitions/dobetterweb.js

```diff
@@ -498,6 +498,33 @@ const expectations = {
           ],
         },
       },
+      'preload-lcp-image': {
+        score: 1,
+        numericValue: 0,
+        details: {
+          items: [{
+            node: {
+              snippet: '<h2 id="toppy" style="background-image:url(\'\');">',
+              nodeLabel: 'Do better web tester page',
+            },
+            url: 'http://localhost:10200/dobetterweb/lighthouse-480x318.jpg?lcp',
```

Comment:
on main, this same updated dbw would have an LCP image URL of the main page, `http://localhost:10200/dobetterweb/dbw_tester.html` instead of the actual image